From aaaae66f03c3543ad15e14e541173bac24cb77b6 Mon Sep 17 00:00:00 2001
From: Xiang Xu <xuxiang@eswincomputing.com>
Date: Mon, 25 Aug 2025 10:11:09 +0800
Subject: [PATCH 518/540] feat:implement pin header function of sbc/vela

Changelogs:
Modify dts in order to implement pin header functions of sbc/vela

Signed-off-by: Xiang Xu <xuxiang@eswincomputing.com>
---
 arch/riscv/boot/dts/eswin/eic7700-sbc-a1.dtsi | 47 ++++---------------
 1 file changed, 8 insertions(+), 39 deletions(-)

diff --git a/arch/riscv/boot/dts/eswin/eic7700-sbc-a1.dtsi b/arch/riscv/boot/dts/eswin/eic7700-sbc-a1.dtsi
index b42aaeda5da30..7b72bbe2cc5da 100755
--- a/arch/riscv/boot/dts/eswin/eic7700-sbc-a1.dtsi
+++ b/arch/riscv/boot/dts/eswin/eic7700-sbc-a1.dtsi
@@ -33,7 +33,7 @@
 #include <dt-bindings/interrupt-controller/irq.h>
 #include <dt-bindings/gpio/gpio.h>
 #include <dt-bindings/input/input.h>
-#include "eic7700-sbc-a1-camera-imx219.dtsi"
+//#include "eic7700-sbc-a1-camera-imx219.dtsi"
 
 /* Clock frequency (in Hz) of the PCB crystal for rtcclk */
 
@@ -365,38 +365,22 @@ &d0_graphcard1 {
 	dais = <&d0_i2s1_port>;
 };
 
-&isp_0 {
-	status = "okay";
-};
-
-&isp_1 {
-	status = "okay";
-};
-
-&dewarp {
-	status = "okay";
-};
-
-&mipi_dphy_rx {
+&vi_subsys {
 	status = "okay";
 };
 
-&csi_dma0 {
-	status = "okay";
+&isp_0 {
+	status = "disabled";
 };
 
-&csi_dma1 {
+&isp_1 {
 	status = "disabled";
 };
 
-&csi2_0 {
+&dewarp {
 	status = "okay";
 };
 
-&csi2_1 {
-	status = "disabled";
-};
-
 &sdhci_emmc {
 	/* unused */
 	status = "disabled";
@@ -454,6 +438,7 @@ &d0_gmac0 {
 	rst-gpios = <&portc 30 GPIO_ACTIVE_LOW>;
 	eswin,rgmiisel = <&pinctrl 0x290 0x3>;
 	eswin,led-cfgs = <0x6100 0xa40 0x420>;
+	phy-mode = "rgmii-txid";
 	dly-param-1000m = <0x23232323 0x800c8023 0x0c0c0c0c>;
 	dly-param-100m = <0x50505050 0x803f8050 0x3f3f3f3f>;
 	dly-param-10m = <0 0 0>;
@@ -614,7 +599,7 @@ es5340@f {
 		compatible = "einno,es5340";
 		reg = <0xf>;
 		eswin,regulator_default-microvolt=<1000000>;
-		eswin,regulator_label = "supply vdd1", "npu vdd1", "npu current1", "npu temperature1";
+		eswin,regulator_label = "Supply VDD", "NPU Voltage", "NPU CURRENT", "NPU Temperature";
 		label = "npu_vdd";
 		regulators{
 			npu_vcc1:npu_svcc{
@@ -733,22 +718,6 @@ &die0_rtc {
 	status = "okay";
 };
 
-&timer0 {
-	status = "okay";
-};
-
-&timer1 {
-	status = "okay";
-};
-
-&timer2 {
-	status = "okay";
-};
-
-&timer3 {
-	status = "okay";
-};
-
 &pinctrl {
 	status = "okay";
 	pinctrl-names = "default";
-- 
2.53.0

